ntoskrnl/io/pnpmgr/plugplay.c

```diff
@@ -245,8 +245,8 @@ IopInitializeDevice(
     }
 
     /* Set the device instance of the device node */
-    RtlDuplicateUnicodeString(0, &DeviceInstance, &DeviceNode->InstancePath);
-
+    StatusUnused = RtlDuplicateUnicodeString(0, &DeviceInstance, &DeviceNode->InstancePath);
+    UNREFERENCED_PARAMETER(StatusUnused);
```

Comment:
Better patch by @Doug-Lyons was:
```suggestion
    Status = RtlDuplicateUnicodeString(0, &DeviceInstance, &DeviceNode->InstancePath);
    if (!NT_SUCCESS(Status))
    {
        DPRINT1("RtlDuplicateUnicodeString() failed (Status 0x%08lx)\n", Status);
        goto done;
    }
```